WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
> But it's really ugly, especially in a public API.

I don't think it's ugly. It sounds like you do, though. One way of dodging this entirey problem problem might be to change it to "attempt value", which is [defined by the WCIF](https://docs.google.com/document/d/1hnzAZizTH0XyGkSYe-PxFL5xpKVWl_cvSdTzlT_kAs8/edit#heading=h.16k58sisi0it).

> So I'd prefer that it's a 2 decimal number, at least in the WCIF. Since JSON is a string format, that shouldn't be that hard.

Are you suggesting making it a 2 decimal number, but encoded as a string? Such as this:

```
{
  "seconds": "1.23"
}
```

versus:

```
{
  "seconds": 1.23
}
```

The former is doable, but I'd argue weird. Implementing the latter would require customizing the JSON parser for every language that wants to read/create WCIF, which I do not think is worth it.